app/models/agents/java_script_agent.rb

```diff
@@ -102,6 +102,7 @@ def execute_js(js_function, incoming_events = [])
           memory.to_json
         end
       end
+      context["unescapeHTML"] = lambda { |a, x| CGI.unescapeHTML(x) }
```

Comment:
Yea, it may not be the right solution.  I'm fine with this addition.  Do you think that we should add `escapeHtml` too?